tests/plugins/example/__init__.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2020 Satoru SATOH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+"""YAML Lint plugin entry point
```

Comment:
The tool is named "yamllint", not "YAML Lint".

Reply:
Thanks for your correction! Applied the fix and pushed it: 8253b1c